### DISCUSSION TITLE
Cannot Query Catalog to difference company (Diference Business Partner Number) connector.

### DISCUSSION THREAD

**Discussion by NSolutionDev** [eclipse-tractusx/tractusx-edc#900] [Q&A] [answered]
I have the issue that cannot query calatog to diference BPN connector.
Then i wonder that is the feature or that is the issue?
Please check the error message below.
Thank you.

DEBUG 2023-12-04T07:47:04.283123558 DSP: Unauthorized: Invalid credential issuer: expected did:web:miw.sample.org:BPNL00000000SMVN, found did:web:miw.sample.org:BPNL0000000003X9

**Comment by NSolutionDev** [comment]
@ndr-brt Do you have any idea on this situation?

**Reply by ndr-brt** [reply]
could you please provide further details? configuration of the two connectors and call made?

**Reply by NSolutionDev** [reply]
Dear Mr. @ndr-brt 

Please check the details as bellow.

This is the architecture! I already add the wallet on MIW of both BPN.
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/20982440/d51d2a5a-2da1-4d46-b185-2c40754bc230)

When query catalog from consumer to provider, the bellow error had return!
![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/20982440/3db02389-f6dd-4fab-b8a5-f7c8a9267297)

Here is the error log on provider:
DEBUG 2023-12-05T04:38:26.096075805 DSP: Incoming catalog request.
SEVERE 2023-12-05T04:38:26.250224176 Invalid credential issuer: expected did:web:miw.***.com:BPNL0000000003X9, found did:web:miw.***.com:BPNL00000000SMVN
DEBUG 2023-12-05T04:38:26.25033466 Unauthorized, [Invalid credential issuer: expected did:web:miw.***.com:BPNL0000000003X9, found did:web:miw.***.com:BPNL00000000SMVN]

Hope that you can detect where is the issue.
Thank you

**Reply by ndr-brt** [reply, marked as answer]
seems like the authority id setting has not be configured correctly on provider side (EDC2).
`tx.ssi.miw.authority.id` should be `BPNL00000000SMVN` put I think currently it is `BPNL0000000003X9`

**Reply by NSolutionDev** [reply]
Thank you very much for your support Mr @ndr-brt. It is resolved.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
